modules/service/src/main/scala/lucuma/odb/graphql/snippet/input/WhereOptionString.scala

```diff
@@ -24,18 +28,20 @@ object WhereOptionString {
         NonEmptyStringBinding.Option("NEQ", rNeq),
         NonEmptyStringBinding.List.Option("IN", rIn),
         NonEmptyStringBinding.List.Option("NIN", rNin),
-        NonEmptyStringBinding.Option("LIKE", rLike),      // checked but unused, for now
-        NonEmptyStringBinding.Option("NLIKE", rNlike),    // same
-        BooleanBinding.Option("MATCH_CASE", rMatchCase)   // same
+        NonEmptyStringBinding.Option("LIKE", rLike),
+        NonEmptyStringBinding.Option("NLIKE", rNlike),
+        BooleanBinding.Option("MATCH_CASE", rMatchCase)
       ) =>
         (rIsNull, rEq, rNeq, rIn, rNin, rLike, rNlike, rMatchCase).mapN {
-          (isNull, eq, neq, in, nin, _, _, _) =>
+          (isNull, eq, neq, in, nin, like, nlike, matchCase) =>
           and(List(
             isNull.map(IsNull(path, _)),
             eq.map(a => Eql(path, Const(a.some))),
             neq.map(a => NEql(path, Const(a.some))),
             in.map(as => In(path, as.map(_.some))),
             nin.map(as => Not(In(path, as.map(a => a.some)))),
+            like.map(s => Like(path, s.value, !matchCase.getOrElse(true))),
```

Comment:
Is this what I expect with `%` and `?` wildcards?

Reply:
Yes, it compiles down to `LIKE` and `ILIKE` and the const argument is passed verbatim.